packages/transformer/src/IModelCloneContext.ts

```diff
@@ -207,6 +207,22 @@ export class IModelCloneContext extends IModelElementCloneContext {
     return targetElementAspectProps;
   }
 
+  /** Clone the specified relationship into RelationshipProps for the target iModel.
+   * @internal
+   */
+  public cloneRelationship(sourceRelationship: Relationship): RelationshipProps {
+    const targetRelationshipProps: RelationshipProps = sourceRelationship.toJSON();
+    targetRelationshipProps.sourceId = this.findTargetElementId(sourceRelationship.sourceId);
+    targetRelationshipProps.targetId = this.findTargetElementId(sourceRelationship.targetId);
+
+    sourceRelationship.forEachProperty((propertyName: string, propertyMetaData: PropertyMetaData) => {
+      if ((PrimitiveTypeCode.Long === propertyMetaData.primitiveType) && ("Id" === propertyMetaData.extendedType)) {
```

Comment:
navigation properties must be remapped as well. Although I forget if the native code makes any attempt at this.

Reply:
actually let's not add cloneRelationships yet... it's too tied to existing work to clone all entities the same way... can you revert this?